log/log.go

```diff
@@ -55,6 +58,7 @@ func CleanExit(format string, v ...interface{}) {
 func Err(format string, v ...interface{}) {
 	fmt.Fprint(Stderr, pretty.Colorize("{{.Red}}[ERROR]{{.Default}} "))
 	fmt.Fprintf(Stderr, appendNewLine(format), v...)
+	ErrorState = true
```

Comment:
We could, and not change any of the current logic from that gist. The select will make it grow nicely later. Could eventually do a HelmError type that has both the OS exit code and a more specific message, but I doubt it's needed for this particular addition.